pom.xml

```diff
@@ -679,6 +684,9 @@
 		<profile>
 			<!-- For deployment where host is local (and ssh isn't available for builder, e.g. CI) -->
 			<id>local-deploy</id>
+			<properties>
+				<hdf5.prefix>/space/opt/hdf5-${hdf5.version}</hdf5.prefix>
+			</properties>
```

Comment:
This is no-longer necessary with our migration to Rocky Linux 9. We should use the system-provided libdf5.